cloudify_aws/codepipeline/resources/pipeline.py

```diff
@@ -101,3 +117,11 @@ def create(ctx, iface, params, **_):
     resource_id = create_response['pipeline']['name']
     iface.update_resource_id(resource_id)
     utils.update_resource_id(ctx.instance, resource_id)
+
+
+@operation
+@decorators.aws_resource(CodePipelinePipeline, RESOURCE_TYPE)
+def execute(ctx, iface, name=None, clientRequestToken=None, **_):
+    execute_response = iface.execute(name, clientRequestToken)
+    ctx.instance.runtime_properties[
+        'execute_pipeline_response'] = execute_response
```

Comment:
Any exceptions we need to handle?

Reply:
Well if it fails i got the client error with detailed message. I can add a catch that raises NonRecoverableError with traceback.